pkg/lib/bundle/supported_resources.go

```diff
@@ -32,6 +33,8 @@ func init() {
 	supportedResources[RoleBindingKind] = true
 	supportedResources[PrometheusRuleKind] = true
 	supportedResources[ServiceMonitorKind] = true
+	supportedResources[SecretKind] = true
+	supportedResources[ConfigMapKind] = true
```

Comment:
nit: can this be a map literal and not in an init?

Reply:
nit: I would also create a named type or some constants for the values here; e.g. `Namespaced, NotNamespaced`.